go/storage/mkvs/checkpoint/restorer.go

```diff
@@ -40,6 +40,22 @@ func (rs *restorer) StartRestore(ctx context.Context, checkpoint *Metadata) erro
 	return nil
 }
 
+func (rs *restorer) AbortRestore(ctx context.Context) error {
+	rs.Lock()
+	defer rs.Unlock()
+
+	if rs.currentCheckpoint == nil {
+		return ErrNoRestoreInProgress
+	}
+
+	rs.pendingChunks = nil
+	rs.currentCheckpoint = nil
+
+	// TODO: also properly remove leftover nodes from the database.
```

Comment:
Open an issue and reference it here.

Reply:
Issue #3241

Reply:
I meant reference it in the above TODO comment.

Reply:
Also nice to have it here tho, for more context.